server.js

```diff
@@ -24,9 +17,116 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+const Animal = mongoose.model("Animal", {
+  name: String,
+  scientificName: String,
+  doesItFart: Boolean,
+  doesItMaybeFart: Boolean,
+  descriptionExists: Boolean,
+  descriptionNotes: String,
+  paperLink: String,
+  verifiedBy: String,
+});
+
+// if (process.env.RESET_DB) {
+// const seedDatabase = async () => {
+//   await Animal.deleteMany();
+//   doItBeFartin.forEach((animal) => {
+//     new Animal(animal).save();
+//   });
+// };
+// seedDatabase();
+// }
+
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next();
+  } else {
+    res.status(503).json({ error: "Service unavailable" });
+  }
+});
+
+// Route explanation
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send([
+    { "/": "Home" },
+    { "/animals": "All the animals" },
+    { "/animal/:animal": "Find one animal" },
+    { "/description/:boolean": "True or false if there a fart description" },
+    { "/doitbefartin/:boolean": "True or false if it be fartin" },
+    {
+      "/doitmaybebefartin/:boolean": "True or false if it MAYBE be fartin",
+    },
+  ]);
+});
```

Comment:
This part is very well structured and easy to understand for the API user, as sometimes the API:s can be a bit tricky to grasp in the landing page.